### DISCUSSION TITLE
Any plans for Pi5 support?

### DISCUSSION THREAD

**Discussion by lupinthief** [mcguirepr89/BirdNET-Pi#1211] [Ideas]
Just wondering whether anyone has this running on Pi5, or is working on it? The ease of adding an M.2 NVME is really appealing as I keep getting SD failures (3B+)

**Comment by alexbelgium** [comment]
Hi, use this fork it has rpi5 support and many updated options: https://github.com/Nachtzuster/BirdNET-Pi

Although he did an incredible job to set it up, the owner of this repo wrote in December 2022 that he would stop supporting the app and did so nearly a year ago.

**Reply by lupinthief** [reply]
Thanks. I'll give it a go!
